src/frontend/shared/components/FriendsPanelItem.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import {IUserProps} from "../interfaces/IUserProps";
+
+class FriendsPanelItem extends React.Component<IUserProps, {}> {
```

Comment:
This class is renamed from `FriendBanner` to `FriendsPanelItem`. Reason: Welcome to name suggestion but I think banner feels like a huge thing that spans across an area, but this component is each individual element inside the side panel. In order to illustrate the relationship, I think `FriendsPanelItem` is better.